packages/react/webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+module.exports = () => {
+  const { WITH_REPORT } = process.env;
+
+  return {
+    mode: 'production',
+    entry: './src/index.ts',
+    output: {
+      path: path.resolve(__dirname, 'dist'),
+      filename: 'logto.react.min.js',
+      library: {
+        type: 'umd',
+        name: 'logto',
+      },
+    },
+    module: {
+      rules: [
+        {
+          exclude: /node_modules/,
+          use: 'ts-loader',
```

Comment:
do we still need `ts-loader` if we already use `tsc` to compile files to js?

Reply:
Removed `Webpack`. Do not support `package` for now and add it when needed.